Tests/Unit/Core/Payment_Method_Factory_Test.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use \Wirecard\Oxid\Core\Payment_Method_Factory;
+
+class Payment_Method_Factory_Test extends OxidEsales\TestingLibrary\UnitTestCase
+{
+    public function testCreatePaypal()
+    {
+        $oPaymentMethod = Payment_Method_Factory::create("wdpaypal");
+        $this->assertTrue($oPaymentMethod instanceof \Wirecard\Oxid\Model\Paypal_Payment_Method);
```

Comment:
Just a minor note: There is also `assertInstanceOf()`.

Reply:
nice!